docs/source/pages/documentation.md

```diff
@@ -1,351 +1,631 @@
-the most useful are the wildcard tag, @*@ and the @TO@ flag.
+The command:
 
-The wildcard flag can be used to avoid specifying particular parts of subject / session names that are wanted to
-to tbe transferred. This is particularly useful for skipping the `date_xxxxxx` flag that might differ across sessions or subjects.
+```
+datashuttle \
+my_first_project \
+upload
+-sub 001
+-ses all_sub
+-dt all_data_type
+```
 
-For example,
-`project.upload_data(sub_names="sub-@*@", ses_names="ses-001_date-@*@", data_type="all")` or
-equivalently `datashuttle my_project upload_data --sub_names sub-@*@ --ses_names ses-001_date-@*@ --data_type all`
+Would upload:
 
-would transfer all any first session, irregardless of date, or all subjects and all data types.
+- Contents residing in the `sub-001` folder only.
+- All *data-type* folder contents (`histology`, `behav`, `ephys`)
 
-## Transferring a specific file or folder
+3) The final example shows the effect of transferring `all_non_sub` files only. The command:
 
-The functions upload_project_folder_or_file() or download_project_folder_or_file() can be used to
-transfer a particular, individual file or folder. The path to the file / folder, either full
-or relative to the project top level folder, should be input.
+```
+datashuttle \
+my_first_project \
+upload
+-sub all_non_sub
+-ses all
+-dt all
+```
+
+Would upload:
+
+- the file `a_project_related_file.json` only.
 
-## Logging
+#### Query DataShuttle for current *configurations*
+
+It is possible to query DataShuttle for the current *configurations* and relevant file paths,
+for example the `show-local-path` command will print the currently set *local* path to the terminal.
+
+Similarly, the command `show-configs` will print all currently set *configurations*. A number of additional convenience functions exist (for example `show-remote-path`, `show-top-level-folder`, `show-config-path`). For a full list, see our API reference [TODO: link].
+
+### Logging
 
 Detailed logs of all configuration changes, folder creation and data transfers are logged
-to a .datashuttle folder in the local project folder. These logs are named
-with the command (e.g. make_config_file), date and time of creation.
+to the `.datashuttle` folder in the that is created in the *local* project folder.
 
-## Convenience Functions
+For each command run, a log of that command is placed in the logs folder, with the time and date of the command. The log itself contains relevant information pertaining to that command. For example, if the commands `make_sub_folders`, `upload`, `download` were run sequentially, the logs output folder would look like:
 
-Convenience functions can be used to quickly get relevant project information. See the API or CLI documentation
-for more information.
+```
+make_sub_folders_2023-06-08_09-55-14.log
+upload_data_2023-06-08_09-55-45.log
+download_data_2023-06-08_09-56-19.log
```

Comment:
Idea for the future. Format log file names as <data_time_command.log>. This way they will be displayed chnonologically in the folder, when alphabetically sorted